### ISSUE TITLE
Subfolders

### ISSUE THREAD

**Issue by Tealk** [toolstack/Folio#11]
Would it be possible to get a function to create subfolders?

**Comment by toolstack** [comment]
That would be more complex, the code base isn't really setup for it, but it might be possible in a future release.

**Comment by PHHENS** [comment]
Could be interesting continuing using Miller Columns (no tree expand/collapse)

**Comment by toolstack** [comment]
Agreed that a hierarchy is needed for the folio's, just not by splitting it out into multiple columns.  I'm thinking more of a tree structure at the moment, but haven't actually started working on anything.

**Comment by PHHENS** [comment]
At my point of view, the millers columns access gets the whole power of folio because of very fast finding and opening  the right file.

**Comment by PHHENS** [comment]
Taking a higher point of view, I think folio could be a more generic app :
- left panel : miller's column file browser (3 or 4 columns with horizontal scrollbar)
- right panel : editor for ascii (txt) and md files, preview for other files type (pdf, html, png, jpg ...)

In brief, a merge of actual folio app and, mac OS finder like, fm app (https://github.com/euclio/fm : try it to see)

Thanks for your understanding.

**Comment by toolstack** [comment]
I don't see Folio going in that kind of direction, the reason I like it is that it's simple and to the point.  It's a MD note taking app... that's it.

So while I agree that subfolder support should be added, I don't think expanding it's feature set beyond what it is makes sense in that light.

**Comment by k8ieone** [comment]
Subfolders would be very nice. That would basically make Folio comparable with tools like Obsidian, only much more lightweight and simple. I would love to see that.

**Comment by PHHENS** [comment]
Could we hope progress on multi level subfolders (or millers columns) in a near future revision ?
Thanks for your nice work ;-)

**Comment by toolstack** [comment]
I played around a bit with an implementation a while ago, but it's no where near ready and no timeline as it's taken a back seat to other things.

**Comment by PHHENS** [comment]
It may be easier to add an option for a third column "group/category of workbooks" ?
This would the number of notes easily ordered by a factor of 10, a could be sufficient for 
many users.

**Comment by toolstack** [comment]
I don't like a third column, so that seems unlikely to be a solution I'd implement.

**Comment by PHHENS** [comment]
Subfolder would really a major improvement in folio.
Any progress in subfolder support ?

**Comment by PHHENS** [comment]
But, if you support just one more level of subfolder, it may be easier to implement ?

**Comment by PHHENS** [comment]
Letter  to Santa  🎅  : Dear Santa, could you please add subfolder support ;-)

**Comment by toolstack** [comment]
You never know, I do have some time off coming up and working on Folio is one of my items on my list 🤷

**Comment by PHHENS** [comment]
Hope you'll find time soon...
You could take a look at BookUp which has a nice column allowing notebook subtrees.

**Comment by PHHENS** [comment]
Any hope to see this feature before next Xmas ;-) ?

**Comment by toolstack** [comment]
I make no commitments on that 😉

I will say that I suspect spell checking is higher on my list at the moment than subfolders, so take that for what it is.
